sonarts-sq-plugin/sonar-typescript-plugin/src/main/resources/org/sonar/l10n/typescript/rules/typescript/Sonar_way_recommended_profile.json

```diff
@@ -92,6 +92,7 @@
     "S4327",
     "S4328",
     "S4335",
-    "S4524"
+    "S4524",
+    "S4624"
```

Comment:
@andrea-guarino-sonarsource why you have new line here? RSPEC doesn't have any profile (while it should!). Fix RSPEC